addons/barcodes/models/barcode_events_mixin.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-from openerp import models, fields, api
+from odoo import api, fields, models
 
 class BarcodeEventsMixin(models.AbstractModel):
```

Comment:
I think you feel to follow PEP8 standard and based on this standard need to take 2 lines between import's section and class.
[GUIDELINE](https://www.python.org/dev/peps/pep-0008/)